### PR TITLE
Update invalid redirect url error code to match our conventions

### PR DESCRIPTION
**Pull request by tom-blake**

### CODE REVIEW

File: src/api/payment-requests/legacy-payment-requests.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ See [Asset Types][] for the list of possible `asset` values for each Asset Type.
 | 403    | {% break _ MERCHANT_CONFIGURATION_NOT_FOUND %} | There was no merchant configuration found for the supplied merchantId and clientId.              |
 | 403    | {% break _ NO_AVAILABLE_PAYMENT_OPTIONS %}     | `asset` is not supported by any of the [Asset Types][] that the [Merchant][] is configured with. |
 | 403    | {% break _ INSUFFICIENT_ASSET_VALUE %}         | The patron does not have enough funds to quick pay the payment request.                          |
+| 403    | {% break _ REDIRECT_URL_INVALID %}             | The supplied redirectUrl does not start with one of the `allowedRedirectUrls` on the [Merchant Config][]. |
```
Comment (tom-blake) [inline src/api/payment-requests/legacy-payment-requests.md:78]: We weren't documenting this error code in the legacy route at all so I've added it here too

Comment (Ray2126) [reply src/api/payment-requests/legacy-payment-requests.md:78]: You can't create a legacy payment request with a redirectUrl

Comment (AndyClifford) [reply src/api/payment-requests/legacy-payment-requests.md:78]: Do we allow redirectUrls in the legacy route?

File: src/api/payment-requests/payment-requests.md [hunk 2]
```diff
@@ -354,7 +354,7 @@ Payment Activities are created when a Payment Request has been **created**, **pa
 | Status |                    Code                    |                                                Description                                                |
 | :----- | :----------------------------------------- | :-------------------------------------------------------------------------------------------------------- |
 | 400    | {% break _ LINE_ITEMS_SUM_CHECK_FAILED %}  | The sum value of the line items did not equal the value of the payment request.                           |
-| 403    | {% break _ RedirectUrl not supported %}    | The supplied redirectUrl does not start with one of the `allowedRedirectUrls` on the [Merchant Config][]. |
+| 403    | {% break _ REDIRECT_URL_INVALID %}         | The supplied redirectUrl does not start with one of the `allowedRedirectUrls` on the [Merchant Config][]. |
```
Comment (Ray2126) [inline src/api/payment-requests/payment-requests.md:357]: Does this break any integrations?

**Review by Ray2126 — CHANGES_REQUESTED** [review]

**Review by ncjones — APPROVED** [review]

**Pull request merged**